packages/fiber/src/three-types.ts

```diff
@@ -14,7 +14,7 @@ export type Matrix4 = THREE.Matrix4 | Parameters<THREE.Matrix4['set']>
 export type Vector2 = THREE.Vector2 | Parameters<THREE.Vector2['set']> | Parameters<THREE.Vector2['setScalar']>[0]
 export type Vector3 = THREE.Vector3 | Parameters<THREE.Vector3['set']> | Parameters<THREE.Vector3['setScalar']>[0]
 export type Vector4 = THREE.Vector4 | Parameters<THREE.Vector4['set']> | Parameters<THREE.Vector4['setScalar']>[0]
-export type Color = THREE.Color | number | string // Parameters<T> will not work here because of multiple function signatures in three.js types
+export type Color = typeof THREE.Color | THREE.Color | number | string // Parameters<T> will not work here because of multiple function signatures in three.js types
```

Comment:
What's the difference between `typeof THREE.Color` & `THREE.Color`?

Reply:
Sorry, it should be `ConstructorParameters<typeof THREE.Color>`